### PR TITLE
Updated polgyonMesh.ts

### PR DESCRIPTION
**Pull request by artfiedler**
Changed the scene to optional and updated the parameter doc

Sorry you guys beat me to merging, looks like one last tiny change here

### CODE REVIEW

File: src/Meshes/polygonMesh.ts [hunk 1]
```diff
@@ -174,10 +174,10 @@ export class PolygonMeshBuilder {
...
      * @param contours Path of the polygon
-     * @param scene scene to add to
+     * @param scene scene to add to when creating the mesh
      * @param earcutInjection can be used to inject your own earcut reference
      */
-    constructor(name: string, contours: Path2 | Vector2[] | any, scene: Scene, earcutInjection = earcut) {
+    constructor(name: string, contours: Path2 | Vector2[] | any, scene?: Scene, earcutInjection = earcut) {
         this.bjsEarcut = earcutInjection;
         this._name = name;
         this._scene = scene;
```
Comment (deltakosh) [inline src/Meshes/polygonMesh.ts]: you should the set it to this._scene = scene || Engine.GetLastCreatedScene()

And then your _scene is not undefined

Comment (artfiedler) [reply src/Meshes/polygonMesh.ts]: even though scene won't be used at all?

**Comment by artfiedler** [comment]
::smh:: git can be seriously annoying sometimes

**Comment by artfiedler** [comment]
scene || Engine.LastCreatedScene

engine.LastCreatedScene ends up being Nullable<Scene> same as | undefined

**Comment by artfiedler** [comment]
hrmm not letting me revert back to before it updated all those files guess ill just update the current state to the previous state that should solve :/

File: src/Meshes/polygonMesh.ts (hunk 1 above)
Comment (deltakosh) [reply src/Meshes/polygonMesh.ts]: yep

**Comment by deltakosh** [comment]
Ok to merge?

**Comment by artfiedler** [comment]
Appears to be

**Pull request merged**